ProjectTemplates/gmodnet-module/.template.config/template.json

```diff
@@ -0,0 +1,14 @@
+﻿{
+	"$schema": "http://json.schemastore.org/template",
+	"author": "SupinePandora43",
```

Comment:
Author must be GmodNET

Reply:
resolved by https://github.com/GmodNET/Templates/pull/2/commits/43b3e391b40db967d94c5f16ea10354c3fa3c28f